.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Pull Request template  
+
+## Description  
+Fixes # (Issue-number)  
```

Comment:
Instead of telling what issue number it fixes, let's keep it generalized. Just add it in brackets like instructing to tell a brief summary of the problem.